scrapers/ar/bills.py

```diff
@@ -138,7 +127,8 @@ def scrape_bill(self, chamber, session):
                 version_url = f"https://www.arkleg.state.ar.us/assembly/{year}/{self.slug}/Bills/{bill_url}.pdf"
             bill.add_version_link(bill_id, version_url, media_type="application/pdf")
 
-            self.scrape_bill_page(bill)
+            for bill_page in self.scrape_bill_page(bill):
+                bill_page
```

Comment:
@jessemortenson scrape_bill_page function is not a simple function. It's a generator because that function has yield.
so a generator isn't run by simple call.

Reply:
Ahhh OK gotcha, thank you. Can you please make this explicit/obvious? Thinking something like:

```
            for bill_page in self.scrape_bill_page(bill):
                # this loop calls the generator until it is done yielding bills
                pass
```

Reply:
Oops - my apologies, I just noticed you already made a change here, thank you - marking resolved